specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/network.json

```diff
@@ -208,6 +207,27 @@
       "required": true,
       "type": "string",
       "description": "Client API version."
+    },
+    "ResourceGroupNameParameter": {
```

Comment:
The parameter 'resourceGroupName' is defined in global parameters section without 'x-ms-parameter-location' extension. This would add the parameter as the client property. Please ensure that this is exactly you want. If so, apply the extension "x-ms-parameter-location": "client". Else, apply the extension "x-ms-parameter-location": "method".

Reply:
Same for VirtualHubNameParameter and ConnectionNameParameter.